src/index.css

```diff
@@ -1050,7 +1044,7 @@ table.table.corpus th:focus {
   .navbar-brand {
     width: 2.2em;
     height: 2.2em;
-    margin: 1em;
+    margin: 1rem;
```

Comment:
Since `em` is a relative unit, margins around the logo got recalculated after reactivating headroom, making mobile navigation bigger than it should be. Falling back to `rem` root size provided a temporary solution. However, I've already simplified margins handling on mobile all together in the new commit.